numpy/_core/tests/test_umath.py

```diff
@@ -1258,6 +1258,17 @@ def test_power_fast_paths(self):
             result = np.power(a, 0.5)
             assert_array_max_ulp(result, expected, maxulp=1)
 
+    def test_large_fast_power(self):
+        # gh-28248
+        for dt in [np.float32, np.float64]:
+            a = np.random.uniform(1, 4, 1000).astype(dt)
+
+            result = np.power(a, 2)
+            assert_array_equal(result, a * a)
+
+            result = np.power(a, 0.5)
+            assert_array_equal(result, np.sqrt(a))
```

Comment:
On non AVX-512 This should help fix the CI failure
```suggestion
            assert_array_max_ulp(result, np.sqrt(a), maxulp=1)
```

Reply:
Sure thanks, that makes sense! Committed your suggestion.